tekton/resources/images/overlays/multi-arch/configuration.yaml

```diff
@@ -0,0 +1,9 @@
+nameReference:
+- kind: TriggerBinding
+  fieldSpecs:
+  - path: spec/triggers/bindings/ref
+    kind: EventListener
+- kind: TriggerTemplate
+  fieldSpecs:
+  - path: spec/triggers/template/name
+    kind: EventListener
```

Comment:
NIT: I've been trying to keep the number of event listeners to a minimum, I feel that having too many services running makes it more difficult to manage. The main reason I kept this event listener separated from the others is that this one uses a service account that can push to the container registry. 
All other CD services are in the same event listener, each matches its own trigger, and the correct trigger is selected using a CEL filter on a dedicated field in the payload sent by the cronjob.

Reply:
So, if I understand everything correct, one EventListener for multi-arch and single-arch cases, different TriggerTemplates and TriggerBindings, use CEL filter to decide which trigger.. to use :) Will work on it